packages/cli-repl/test/e2e.spec.ts

```diff
@@ -336,7 +336,10 @@ describe('e2e', function() {
       shell.assertNoErrors();
     });
 
-    it('rewrites async properly for mapReduce', async() => {
+    it('rewrites async properly for mapReduce', async function() {
+      if (process.env.MONGOSH_TEST_FORCE_API_STRICT) {
+        return this.skip(); // mapReduce is unversioned
+      }
```

Comment:
We could replace these with `skipIfApiStrict()` but that's a nit IMHO :)

Reply:
Yeah, I just wanted to avoid the extra nesting here :)